geojson/geometry.py

```diff
@@ -53,6 +53,13 @@ def __init__(self, geometries=None, **extra):
         super(GeometryCollection, self).__init__(**extra)
         self["geometries"] = geometries or []
 
+    def errors(self):
+        return [geom.errors() for geom in self['geometries']]
```

Comment:
I promise this is my last comment! 🙈 

I noticed that for [`MultiPoint`](https://github.com/frewsxcv/python-geojson/blob/14f85eebc11776f30bb88577b79956386cffbdba/geojson/geometry.py#L71), [`MultiLineString`](https://github.com/frewsxcv/python-geojson/blob/14f85eebc11776f30bb88577b79956386cffbdba/geojson/geometry.py#L87), and [`MultiPolygon`](https://github.com/frewsxcv/python-geojson/blob/14f85eebc11776f30bb88577b79956386cffbdba/geojson/geometry.py#L107), we call [`check_list_errors`](https://github.com/frewsxcv/python-geojson/blob/14f85eebc11776f30bb88577b79956386cffbdba/geojson/base.py#L133-L137) which:

1. iterates over all the sub-geometries
1. checks for errors for each subgeometry
1. if there's no error, no item gets added to the resulting list

This behavior is different from your `GeometryCollection` behavior where the result here be something like `[[], [], []]`.

What do you think? Would it be good to make the implementations more similar or do you think we should keep them how they are right now?

Reply:
No problem. I think it is a slightly different case than the Multi geometries because a geometry collection could contain any of those objects. However I do like the idea of consistency around errors array, so i'll change the method to only add the errors if they exist. Then we can leave the baseclass is_valid property.